### DISCUSSION TITLE
Where can I find sources for https://aspnet.github.io/quickgridsamples/ website?

### DISCUSSION THREAD

**Discussion by sergiy-sc** [dotnet/aspnetcore#49532] [General] [answered]
I couldn't figure out how it's made. It doesn't seem to load webassembly runtime nor it's not blazor server. Are the sources available for that website?

Thanks.

**Comment by gfoidl** [comment, marked as answer]
See https://github.com/dotnet/aspnetcore/issues/47261 (and a 👍🏻 to that issue, so it may get higher priorization).

**Comment by sergiy-sc** [comment]
thanks Günther!
